src/dataset.ts

```diff
@@ -119,6 +119,11 @@ export interface Structure {
      * expressed in Angströms.
      */
     cell?: number[];
+    /**
+     * bonds in structures given as `[id1 id2 order]`, where
+     * **id1**, **id2**, and **order** are integers.
+     */
+    bonds?: Array<Array<number>>;
```

Comment:
I think this should work as well and convey the intent more clearly

```suggestion
    bonds?: Array<[number, number, number]>;
```